### PR TITLE
#983 breadcrumb issues

### PR DESCRIPTION
**Pull request by tbl73**

### CODE REVIEW

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -35,6 +36,13 @@ def get_data
     logger.debug("DEBUG: #{current_user.inspect}")
   end
 
+  def remove_col_id
```
Comment (benwbrum) [inline app/controllers/dashboard_controller.rb]: Can you move the definition `remove_col_id` to `ApplicationController` and remove some of the repeated code?

File: app/controllers/user_controller.rb [hunk 2]
```diff
@@ -8,6 +8,13 @@ def demo
     redirect_to dashboard_path
   end
 
+  def remove_col_id
```
Comment (benwbrum) [inline app/controllers/user_controller.rb]: Move to `ApplicationController`

File: app/controllers/deed_controller.rb [hunk 3]
```diff
@@ -3,6 +3,10 @@ class DeedController < ApplicationController
   PAGES_PER_SCREEN = 50
 
   def list
+    #get rid of col_id if no breadcrumbs
```
Comment (benwbrum) [inline app/controllers/deed_controller.rb]: Call `remove_col_id` from `ApplicationController`

**Comment by tbl73** [comment]
I just pushed up the refactored code.

**Pull request merged**